### ISSUE TITLE
Importing a channel via the CLI (with "force") does not overwrite correctly

### ISSUE THREAD

**Issue by rbeckman-nextgen** [nextgenhealthcare/connect#3126]
When you import a channel (with "force") and the channel has the same name, the currently existing channel is overwritten. However when the channel has the same ID as a currently existing channel but a different name, it is **not** being overwritten.

Imported Issue. Original Details:
Jira Issue Key: MIRTH-3214
Reporter: narupley
Created: 2014-04-01T14:03:11.000-0700

**Comment by rbeckman-nextgen** [comment]
Per Narupley, if a channel is force imported and has the same ID of one channel and the same name of another channel, it should overwrite the channel with the same name and generate a new ID. 

Imported Comment. Original Details:
Author: leilanim
Created: 2014-05-22T09:12:41.000-0700

**Comment by rbeckman-nextgen** [comment]
Force importing a channel with an existing ID will now overwrite the original channel. If the imported channel contains the same name of an existing channel and the same ID as another, the channel with the matching name will be overwritten. 

Imported Comment. Original Details:
Author: leilanim
Created: 2015-07-23T13:36:48.000-0700

**Comment by rbeckman-nextgen** [comment]
Rev 7796
The behavior I saw is below. This looks good according to the description/comments. 

Import without force:
Same id and name - New channel is created with a random uuid as name and id.
Same name different id - New channel is created with a random uuid as name but it keeps the id from import xml. 
Same id different name - New channel is created with new name and id. 
Same id different name(existing) - New channel is created with random uuid as name and id. 

Import with force:
Same id and name - Channel is overwritten. 
Same name different id - Channel with matching name is overwritten, even if id matches another channel. Channel keeps existing id in mirth connect. 
Same id different name - Channel with same id is overwritten. It is given the name from the xml in mirth connect and id remains the same. 
Same id different name(existing) - The channel with the matching NAME is the one that is overwritten. The channel that is overwritten keeps its id, the first comment says otherwise but we confirmed that this is behavior is consistent. 

Imported Comment. Original Details:
Author: eduardoa
Created: 2015-09-10T15:26:03.000-0700
